README.md

```diff
@@ -12,6 +12,18 @@ Repo for you to raise a Pull Request for practice.
 
 Any questions please ask, by raising an `Issue`.
 
+## How to clone this repo and making changes locally
+
+1. click on the clone button(green in colour)
+2. Using your git on your local machine
+3. Type git clone https://github.com/eddiejaoude/hacktoberfest-practice.git
```

Comment:
can the command be put inside inline code using backtics 

For example

- type `git ...`

Reply:
👍